systemtest/src/main/resources/log4j2.properties

```
@@ -15,7 +14,7 @@ appender.rolling.policies.size.size=100MB
 appender.rolling.strategy.type = DefaultRolloverStrategy
 appender.rolling.strategy.max = 5
 appender.rolling.layout.type = PatternLayout
-appender.rolling.layout.pattern=%d{yyyy-MM-dd HH:mm:ss}{GMT} %-5p [%c{1}:%L] %m%n
+appender.rolling.layout.pattern=%d{yyyy-MM-dd HH:mm:ss}{GMT} [T-%3.3T] %notEmpty{[%X{testClass}]}%notEmpty{[%X{testMethod}]} %highlight{%-5p} [%c{1}:%L] %m%n
```

```suggestion
appender.rolling.layout.pattern=%d{yyyy-MM-dd HH:mm:ss}{GMT} [T-%.3T] %notEmpty{[%X{testClass}]}%notEmpty{[%X{testMethod}]} %highlight{%-5p} [%c{1}:%L] %m%n
```

I will commit this as we agree on exact format.